app/src/androidTest/java/ch/epfl/sdp/blindwar/MainMenuActivityTest.kt

```diff
@@ -90,6 +89,9 @@ class MainMenuActivityTest : TestCase() {
         Thread.sleep(1000)
         onView(ViewMatchers.withId(R.id.profileButton))
             .perform(ViewActions.click())
+        onView(ViewMatchers.withId(R.id.emailView))
+            .perform(ViewActions.closeSoftKeyboard())
+        Thread.sleep(1000)

```

Comment:
I think that you don't need to sleep and you can just write closeSoftKeyboard out of a view

Reply:
I hope it works tho

Reply:
Bruh I've slept like 2 nights this week